legend-engine-xt-relationalStore-mutation-executionPlan-test/src/main/resources/core_relational_mutation_execution_test/relational/mutation/testRelationalMutation.pure

```diff
@@ -0,0 +1,299 @@
+    ' kerberos  : String[1];\n'+
+    ' firstName : String[1];\n'+
+    ' lastName  : String[1];\n'+
+    ' firm      : meta::relational::mutation::executionPlan::test::Firm[1];\n'+
+    '}\n'+
+    '\n'+
+    'Class meta::relational::mutation::executionPlan::test::Firm\n'+
+    '{\n'+
+    ' name              : String[1];\n'+
+    ' numberOfEmployees : Integer[1];\n'+
+    '}\n'+
+    '\n'+
+    'function meta::relational::mutation::executionPlan::test::completeGraphFetchTreeH2PersonMutation(jsonString: String[1]): meta::relational::mutation::executionPlan::test::Person[*]\n'+
+    '{\n'+
+    '  let person = meta::relational::mutation::executionPlan::test::Person->internalize(meta::relational::mutation::executionPlan::test::Person_Binding, $jsonString);\n'+
+    '  meta::relational::mutation::executionPlan::test::Person->meta::pure::mutation::execution::save(#{meta::relational::mutation::executionPlan::test::Person{firstName, lastName, kerberos, firm}}#, $person, meta::relational::mutation::executionPlan::test::Person_Mapping, meta::relational::mutation::executionPlan::test::H2Runtime);\n'+
+    '}\n'+
+    '\n'+
+    'function meta::relational::mutation::executionPlan::test::completeGraphFetchTreeMemSQLPersonMutation(jsonString: String[1]): meta::relational::mutation::executionPlan::test::Person[*]\n'+
+    '{\n'+
+    '  let person = meta::relational::mutation::executionPlan::test::Person->internalize(meta::relational::mutation::executionPlan::test::Person_Binding, $jsonString);\n'+
+    '  meta::relational::mutation::executionPlan::test::Person->meta::pure::mutation::execution::save(#{meta::relational::mutation::executionPlan::test::Person{firstName, lastName, kerberos, firm}}#, $person, meta::relational::mutation::executionPlan::test::Person_Mapping, meta::relational::mutation::executionPlan::test::MemSQLRuntime);\n'+
+    '}\n'+
+    '\n'+
+    'function meta::relational::mutation::executionPlan::test::incompleteGraphFetchTreeH2PersonMutation(jsonString: String[1]): meta::relational::mutation::executionPlan::test::Person[*]\n'+
+    '{\n'+
+    '  let person = meta::relational::mutation::executionPlan::test::Person->internalize(meta::relational::mutation::executionPlan::test::Person_Binding, $jsonString);\n'+
+    '  meta::relational::mutation::executionPlan::test::Person->meta::pure::mutation::execution::save(#{meta::relational::mutation::executionPlan::test::Person{firstName, kerberos, firm}}#, $person, meta::relational::mutation::executionPlan::test::Person_Mapping, meta::relational::mutation::executionPlan::test::H2Runtime);\n'+
+    '}\n'+
+    '\n'+
+    'function meta::relational::mutation::executionPlan::test::incompleteGraphFetchTreeMemSQLPersonMutation(jsonString: String[1]): meta::relational::mutation::executionPlan::test::Person[*]\n'+
+    '{\n'+
+    '  let person = meta::relational::mutation::executionPlan::test::Person->internalize(meta::relational::mutation::executionPlan::test::Person_Binding, $jsonString);\n'+

```

Comment:
what's you view on this api

```
meta::relational::mutation::executionPlan::test::Person->internalize(meta::relational::mutation::executionPlan::test::Person_Binding, $jsonString)->meta::pure::mutation::execution::save(#{meta::relational::mutation::executionPlan::test::Person{firstName, kerberos, firm}}#, meta::relational::mutation::executionPlan::test::Person_Mapping, meta::relational::mutation::executionPlan::test::MemSQLRuntime);
```

seems easier

Reply:
Updated.